presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDriverUri.java

```diff
@@ -268,6 +269,31 @@ public void testUriWithCustomHeaders()
         assertEquals(properties.getProperty(CUSTOM_HEADERS.getKey()), customHeaders);
     }
 
+    @Test
+    public void testUriWithClientTags()
+            throws SQLException
+    {
+        String clientTags = "c1,c2";
+        PrestoDriverUri parameters = createDriverUri("presto://localhost:8080?clientTags=" + clientTags);
+        Properties properties = parameters.getProperties();
+        assertEquals(properties.getProperty(CLIENT_TAGS.getKey()), clientTags);
+    }
+
+    @Test
+    public void assertNonAlphanumericClientTags()
+    {
+        try {
+            String clientTags = "d1,d2,d3";
+            PrestoDriverUri parameters = createDriverUri("presto://localhost:8080?clientTags=" + clientTags);
+            Properties properties = parameters.getProperties();
+            assertEquals(properties.getProperty(CLIENT_TAGS.getKey()), clientTags);
+        }
+        catch (SQLException e) {
```

Comment:
Nit: better to use the annotation--`@Test(expectedExceptions = ...`

Reply:
@tdcmeehan okay, will squash my commits and add the annotation as you have suggested.